tools/proofers/preview.js

```diff
@@ -938,32 +952,33 @@ const makePreview = function (txt, viewMode, wrapMode, styler, getMessage) {
             if (viewMode !== "flat") {
                 classes.push(formatClass);
             }
-            return (classes.length > 0) ? ` class='${classes.join(" ")}'` : '';
+            return classes.length > 0 ? ` class='${classes.join(" ")}'` : "";
         }
 
         function spanStyle(match, p1, p2) {
-        // p1 is "/" or "", p2 is the tag id
+            // p1 is "/" or "", p2 is the tag id
             var tagMap = {
-                "i": "_",
-                "b": "=",
-                "f": "~",
-                "g": "$",
-                "sc": "",
-                "u": "%"
+                i: "_",
```

Comment:
In Javascript, object property names can be any string (or symbol), but you can only use the dot unquoted property names if they are a valid javascript identifier
https://developer.mozilla.org/en-US/docs/Glossary/Identifier

Since all of these entries in the object are valid javascript identifiers, we don't need to quote them here. Prettier is of the opinion that if you don't need to quote, you shouldn't quote. Just one of the sides of using an opinionated formatter I suppose.